crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -231,7 +241,6 @@ impl ConsensusStatsAPI for ConsensusStatsV1 {
 #[derive(Serialize, Deserialize, Clone, Debug, Default, PartialEq, Eq)]
 pub struct ExecutionIndicesWithStats {
     pub index: ExecutionIndices,
-    pub hash: u64,
```

Comment:
Should we keep this for db compatibility as well?

Reply:
ah, didn't notice they were both in the db